api/src/routing.js

```diff
@@ -158,7 +159,8 @@ if (process.argv.slice(2).includes('--allow-cors')) {
 
 app.use((req, res, next) => {
   req.id = uuid.v4();
```

Comment:
A shower thought I had this morning... a UUID is IMO overkill for this. It doesn't matter if very occasionally there's a conflict, or even if we reuse IDs from week to week, right? The downside of such overkill is adding 16k to every log line, which we were already doing in API but now also doing in the audit log. Again maybe this is a separate issue but let's find a shorter alternative...

Reply:
I know, I thought about this as well. uuids are really convenient, we could just truncate the uuid to 1/4?

Reply:
I was thinking hash it rather than truncate because it feels randomer but yeah, basically! There's packages to do this ( https://www.npmjs.com/package/short-uuid ) but if it's as simple as I expect then we can just do our own implementation.

Reply:
as far as I can tell, `short-uuid` just shortens the uuid to a specific length. I wouldn't bother hashing and adding a potentially complex operation.

Reply:
I added a 12 character limit to request ids.

Reply:
Did you choose 12 because the 13th character is always a 4? Because that's what I would've done!

Anyway, nice work, quick win.

Reply:
hahaha

Actually, I chose 12 because docker containers have unique 12 char identifiers.